tests/db_engine_specs/bigquery_tests.py

```diff
@@ -33,18 +41,26 @@ def test_bigquery_sqla_column_label(self):
             self.assertEqual(actual, expected)
 
     def test_convert_dttm(self):
+        """
+        DB Eng Specs (bigquery): Test conversion to date time
+        """
         dttm = self.get_dttm()
         test_cases = {
             "DATE": "CAST('2019-01-02' AS DATE)",
             "DATETIME": "CAST('2019-01-02T03:04:05.678900' AS DATETIME)",
             "TIMESTAMP": "CAST('2019-01-02T03:04:05.678900' AS TIMESTAMP)",
+            "TIME": "CAST('03:04:05.678900' AS TIME)",
+            "TIMEDATE": None,
```

Comment:
I was thrown off for a second, "what type is that?" 😆 Perhaps call it `UNKNOWNTYPE` to make it clear that it's not expected to be found.

Reply:
sure